Sources/Identity.swift

```diff
@@ -102,4 +137,41 @@ import Foundation
             createXDMSharedState(data: state.identityProperties.toXdmData(), event: event)
         }
     }
+
+    // MARK: GetUrlVariables helpers
+
+    /// Helper function to generate url variables in format acceptable by the AEP web SDK
+    static func generateURLVariablesPayload(configSharedState: [String: Any], identityProperties: IdentityProperties) -> String {
```

Comment:
Can you move this and the helper function to a separate class `URLAppender`. I think that will improve readability and maintainability considering AEPIdentity direct does the same.